CHANGELOG.md

```diff
@@ -5,10 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [unreleased]
+
+### Added
+- Added XCP-style quality control file
+- Added `engine.log` (when verbose debugging is on)
+- Added ability to fix random seed for
```

Comment:
Perhaps @sgiavasis can provide a quick scan to see if there other interfaces where we urgently would want to fix random seed, otherwise we can just grow this list as-needed.